openmc/mgxs_library.py

```diff
@@ -908,10 +935,12 @@ def set_multiplicity_mgxs(self, nuscatter, scatter, nuclide='total',
                     ['universe', 'cell', 'material'])
 
         if self._representation is 'isotropic':
+            # import pdb; pdb.set_trace()
```

Comment:
Oh shoot, thanks.

Yes, I have.  I've just gotten used to the commands for GDB (which are the same for PDB generally) and so its not really a big deal to use the terminal commands.